app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V1
+    class ItemsController < ApplicationController
+      def index
+        Item.all
+        render json: ItemSerializer.new(airlines).serialized_json
```

Comment:
Hello, @r-ahmed2022 
Should `airlines` be `item`?

Reply:
done, as advised